auth0/resource_auth0_organization_member_test.go

```diff
@@ -0,0 +1,187 @@
+package auth0
+
+import (
+	"log"
+	"strings"
+	"testing"
+
+	"github.com/auth0/go-auth0/management"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/auth0/terraform-provider-auth0/auth0/internal/template"
+)
+
+func init() {
+	resource.AddTestSweepers("auth0_organization_member", &resource.Sweeper{
+		Name: "auth0_organization_member",
+		F: func(_ string) error {
+			api, err := Auth0()
+			if err != nil {
+				return err
+			}
+
+			var page int
+			var result *multierror.Error
+			for {
+				organizationList, err := api.Organization.List(management.Page(page))
+				if err != nil {
+					result = multierror.Append(result, err)
+					break
+				}
+
+				for _, organization := range organizationList.Organizations {
+					log.Printf("[DEBUG] ➝ %s", organization.GetName())
+
+					if strings.Contains(organization.GetName(), "test") {
+						result = multierror.Append(
+							result,
+							api.Organization.Delete(organization.GetID()),
+						)
+
+						log.Printf("[DEBUG] ✗ %s", organization.GetName())
+					}
+				}
+				if !organizationList.HasNext() {
+					break
+				}
+				page++
+			}
+
+			users, err := api.User.List()
+			if err != nil {
+				return err
+			}
+			for _, user := range users.Users {
+				if strings.Contains(*user.Email, "test") {
+					err = api.User.Delete(*user.ID)
+					result = multierror.Append(result, err)
+				}
+			}
+
+			roles, err := api.Role.List()
+			if err != nil {
+				return err
+			}
+			for _, role := range roles.Roles {
+				if strings.Contains(*role.Name, "test") {
+					err = api.Role.Delete(*role.ID)
+					result = multierror.Append(result, err)
+				}
+			}
+
+			return result.ErrorOrNil()
+		},
+	})
+}
+
+func TestAccOrganizationMember(t *testing.T) {
+	httpRecorder := configureHTTPRecorder(t)
+
+	testName := strings.ToLower(t.Name())
+
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: testProviders(httpRecorder),
+		Steps: []resource.TestStep{{
+			Config: template.ParseTestName(testAccOrganizationMembersAux+`
```

Comment:
I would prefer if we put the config in a separate const above the test function, so it's easier to just check the assertions in here.

Reply:
I find that keeping the configuration and the assertions close together allows for easier comparison and rids us the need to devise obtuse naming conventions.

Reply:
Okay, I guess it can be helpful for these tests, but if the resource is a lot more customizable the test assertions end up being polluted.